main.js

```diff
@@ -1192,13 +1200,26 @@ require(['jquery', 'underscore', 'utils', 'video', 'soundchip', 'ddnoise', 'debu
                 last = 0;
                 return;
             }
-            requestAnimationFrame(draw);
+            var speedy;
```

Comment:
maybe rephrase as:

```javascript
var speedy = fastAsPossible || (fastTape && processor.acia.motorOn);
if (speedy) { ... }
```

Seeing an uninitialised variable tickles my C++ spider sense (even though it's totally fine in this context :))

Reply:
Fixed in latest upload.